xcodeproj/internal/templates/installer.sh

```diff
@@ -147,7 +147,6 @@ rsync \
   --chmod=u+w,F-x \
   --exclude=project.xcworkspace \
   --exclude=rules_xcodeproj/bazel \
-  --exclude=xcuserdata \
```

Comment:
@brentleyjones I forget—will this force an Xcode project change prompt?

Reply:
I believe it's just `project.xcworkspace`: https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/570175fb14305d44062a15b660d659f2fa4853e7/xcodeproj/internal/templates/installer.sh#L143